runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,133 @@
+{
+    "ociVersion": "1.0.1",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "args": [
+            "/firecracker",
+            "--api-sock",
+            "api.socket"
+        ],
+        "env": [
+            "PATH=/"
+        ],
+        "cwd": "/",
+        "capabilities": {
+            "effective": [
+            ],
+            "bounding": [
+            ],
+            "inheritable": [
+                "CAP_NET_ADMIN"
```

Comment:
Do we still need `CAP_NET_ADMIN`?

Reply:
Yea, you will get an error otherwise :/

Reply:
What error?  Firecracker should not require `CAP_NET_ADMIN`.

Reply:
```
&{FaultMessage:Cannot open TAP device. Invalid name/permissions. CreateTap(Os { code: 1, kind: PermissionDenied, message: Operation not permitted })}
```

Reply:
I'm not 100% sure, but I suspect this requirement comes from the fact that the tap device owner/group is not being set to the uid/gid being configured with the jailer.

For statically configured network interfaces, it's up to the user to set the owner/group on the tap (as the user is expected to manually pre-create the tap device).

For CNI configured network interfaces, it's up to the CNI plugin. For `tc-redirect-tap` specifically, there is [an open issue to support this](https://github.com/firecracker-microvm/firecracker-go-sdk/issues/132).

If the tap uid/gid is in fact the only issue here, then I don't think there's a hard requirement for Firecracker to have `CAP_NET_ADMIN`, it's conditional depending on their network setup. I guess the ideal approach in my mind would be that we default to not having the capability but give the user a way of optionally adding it. I'd be okay for that to be a followup if it's too much to update this PR with that though.

Reply:
I would very much prefer to not have `CAP_NET_ADMIN` granted to the Firecracker process.  I'm okay with this being in a followup PR if you don't think you can address it now, but we should be able to run Firecracker with zero capabilities.

Reply:
Yea, let me verify that it is only the UID and GID issue, before proceeding. If it turns out that is the only issue, then I don't think it'll be too hard to fix.